content/applications/general/calendars/google/google_calendar_credentials.rst

```diff
@@ -45,58 +46,57 @@ Calendar API` from the search results. Click :guilabel:`Enable`.
    :align: center
    :alt: Enable the Google Calendar API.
 
-Create credentials
-------------------
+OAuth consent screen
+--------------------
 
-Now that you have created your API project and enabled the Google Calendar API, you need to create
-credentials. Begin by clicking :guilabel:`Create Credentials`. Google will then guide you through
-four steps to create your API credentials.
+Now that the API project has been created OAuth should be configured. Click on :guilabel:`OAuth
+consent` in the left menu and then select the :guilabel:`User Type`.
```

Comment:
Now that the API project has been created, OAuth should be configured. To do that, click on :guilabel:`OAuth
consent` in the left menu, and then select the :guilabel:`User Type`.